sbol3/validation.py

```diff
@@ -50,6 +50,10 @@ def __iter__(self):
         for warning in self._warnings:
             yield warning
 
+    def __str__(self):
+        xs = self._errors + self._warnings
+        return '\n'.join([e.__str__() for e in xs])
```

Comment:
I'd like to see this call `str(e)` instead of `e.__str__()`. I think the former is much clearer and pythonic.

```suggestion
        return '\n'.join([str(e) for e in xs])
```